NEWS.md

```diff
@@ -1,5 +1,17 @@
 ## 13.8.0 - Unreleased
 
+## 13.7.1 - Released (Quesnelia R1 2024)
+This release focused on adding ability to search by location and holding in POL  
+[Full Changelog](https://github.com/folio-org/mod-orders-storage/compare/v13.7.0...v13.7.1)
+
+### Stories
+* [MODORDSTOR-396](https://folio-org.atlassian.net/browse/MODORDSTOR-396) - Add indexes to satisfy filtering by metadata fields
```

Comment:
I will rename this story later, lets rename in News to "Add index to satisfy filtering by searchLocationIds"